app/views/users/show.html.slim

```diff
@@ -46,6 +46,12 @@
         | :
       .col.row-value
         = @user.email
+    .row
+      .col-auto.row-label
+        = t('.openai_api_key')
+        | :
+      .col.row-value
+        = @user.openai_api_key
```

Comment:
```suggestion
      .col.row-value
        - if @user.openai_api_key.present?
          = 'entered'
        - else
          = 'not entered'
```

Please use translations and try to think of better word than (not) entered.

Reply:
Does `Vorgesehen` and `Nicht Vorgesehen` work?

Reply:
"Vorgesehen" does not fit here.
Maybe "entered" and "eingegeben" would be fine. I can't really think of better alternatives

Reply:
Is there a translation for `provided` that fits?

Reply:
Maybe `Inserted` and `Not inserted`? what would be a good german translation for that?

Reply:
`Inserted` as well as `Provided` makes it sound like something that the system provides and not the user. 
I'd go for `Entered` / `Not entered` or `(hidden)` / `Not entered`

Reply:
Understood. Done and pushed. 

Everything should be tackled now ✌🏼